Drivers/Src/MotorController.c

```diff
@@ -89,36 +178,66 @@ ErrorStatus MotorController_Read(CANbuff *message, car_state_t *car){
             // If we're reading the output from the Motor Status command (0x241) then 
             // Check the status bits we care about and set flags accordingly
             case MOTOR_STATUS: {
-                if(MASK_MOTOR_TEMP_ERR & firstSum)
-                {
-                    car->MotorErrorCode.motorTempErr = ON;
-                }
-
-                if(MASK_SS_ERR & firstSum)
-                {
-                    car->MotorErrorCode.slipSpeedErr = ON;
-                }
-
-                if(MASK_CC_ERR & firstSum)
-                {
-                    car->MotorErrorCode.CCVelocityErr = ON;
-                }
-
-                if(MASK_OVER_SPEED_ERR & firstSum)
-                {
-                    car->MotorErrorCode.overSpeedErr = ON;
-                }
+                if(MASK_MOTOR_TEMP_ERR & firstSum) Motor_FaultBitmap |= T_TEMP_ERR;
```

Comment:
Personally, I prefer the newline and curly braces for if statements, even if there's just one line because I think it's easier to read